File: leetCode/heeseok/344. Reverse String.py
from typing import List

class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        sSize = len(s)
        for i in range(0, int(sSize / 2)):
            leftIndex = i
            rightIndex = sSize - (1 + i)
            tmp = s[leftIndex]
            s[leftIndex] = s[rightIndex]
            s[rightIndex] = tmp

File: leetCode/jio/344. Reverse String.java
class Solution {
    public void reverseString(char[] s) {
        
        int size = s.length;
        int j = size/2;
        char temp;
        
        for(int i = 1; i <= j ; i++){
           
            temp = s[size-i]; 
            s[size-i] = s[i-1];
            s[i-1] = temp;
        }
    }
}
